src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+BASEDIR="/usr/bin"
+
+# This script is used to flash the Beam OS 2 to the SD-Card.
+# Then the Mr Beam will start to Beam OS 2.
+
+# This script uses migration.sh script
+
+# Usage:
+#   Flash the SD-Card
+#   Mount the SD-Card
+#   Restore Sensitive Data
+#   Set LED Status Success/Fail
+#   Shutdown the device
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+timestamp()
+{
+ date +"%Y-%m-%d %T"
+}
+
+do_exit()
+{
+  RET_CODE="$?"
+  echo "$(timestamp) $0: Exitting. ${RET_CODE}"
+  if [ "$RET_CODE" -eq 0 ]; then
+    echo "$0: Normal exiting."
+  else
+    echo "$0: Exiting with error code [${RET_CODE}]"
+    set_status_fail
```

Comment:
This should be called to sh set status fail correct?

Reply:
oh, yes, it should. uptating..